packages/common-all/src/abTests.ts

```diff
@@ -65,49 +65,30 @@ export const GRAPH_THEME_FEATURE_SHOWCASE_TEST = new ABTest(
   ]
 );
 
-/**
- *
- * These enums correspond to the alt treatment hierarchy, and will have to match
- * in order to be synced correctly / loaded by the tutorial initializer.
- *
- * e.g.)
- *  1. in `dendron-site`, prepare tutorials in `tutorial.main.*`, `tutorial.my-treatment-1.*`, and `tutorial.my-treatment-2.*`,
- *  2. in this case, the enum kvps should match the second level hierarchy:
- *     export enum TutorialTestGroups {
- *       "main" = "main",
- *       "my-treatment-1" = "my-treatment-1",
- *       "my-treatment-2" = "my-treatment-2"
- *     }
- */
-export enum TutorialTestGroups {
-  "main" = "main", // shows main tutorial
-  "no-refactor" = "no-refactor", // shows main tutorial, minus mentions to refactoring
-  "original" = "original", // shows old (origianl) tutorial
+export enum QuickstartTutorialTestGroups {
+  "main" = "main",
+  "quickstart-v1" = "quickstart-v1",
 }
 
 /**
- * Long running A/B test for differently treated tutorials.
```

Comment:
fyi: I'm removing these since we decided (@jonathanyeung pointed out) that we shouldn't keep a long-running A/B test group that changes over time.